application/src/main/java/org/togetherjava/tjbot/commands/generic/PingCommand.java

```diff
@@ -0,0 +1,35 @@
+package org.togetherjava.tjbot.commands.generic;
+
+import net.dv8tion.jda.api.events.interaction.SlashCommandEvent;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.togetherjava.tjbot.commands.example.AbstractCommand;
+
+/**
+ * Ping Pong command.
+ */
+public class PingCommand extends AbstractCommand {
+    private static final Logger logger = LoggerFactory.getLogger(PingCommand.class);
+
+    public PingCommand() {
+        super("Ping Pong",
```

Comment:
Again, no spaces, the commands name should be `ping` so the user runs `/ping`

Reply:
I didn't understand that the command name here would actually be the actual string that represents the command, and that the user calls.

We should add some javadoc to it explaining that

Reply:
Has been PR'd #119, but need to fix some things before that will be merged.